src/main/java/Deadline.java

```diff
@@ -0,0 +1,49 @@
+import java.time.LocalDateTime;
+import java.time.DateTimeException;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Deadline extends Task {
+    protected LocalDateTime byDateTime;

```

Comment:
Would names like "deadline" be better? I was a little bit puzzled by the name haha.

Reply:
Implemented. Appreciate for the feedback! 😄